.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 .DS_Store
 _site
 .sass-cache/
+.jekyll-metadata
```

Comment:
:ok_hand:

Reply:
So glad I could contribute... I liked that I was able to see a use-case for docker and thanks for helping out too.

Reply:
and I'm glad that asking contributors to run docker wasn't more complex to setup Ruby as I feared.